packages/react/src/components/List/List.mdx

```diff
@@ -29,29 +29,93 @@ import { List } from '@digdir/designsystemet-react';
 </List.Root>;
 ```
 
-## Eksempler på bruk av `List`
+## Varianter av `List`
 
-### Uten heading
+<br />
+
+### Unummerert liste
+
+I unummererte lister bruker vi kulepunkter. Hensikten med en slik liste er å organisere informasjonen og fremheve viktige opplysninger. Her er ikke rekkefølgen skal utføre noe viktig, men vi må likevel passe på at vi presenterer det innholdet som er mest relevant først.
+
+<Canvas of={ListStories.Unummerert} />
+
+### Nummert liste
+
+En nummerert liste brukes når det er viktig at brukerne forstår grader av viktighet, et hendelsesforløp eller lignende. En nummerert liste egner seg f.eks. for de ulike trinnene i matlaging.
+
+<Canvas of={ListStories.Nummerert} />
+
+<br />
+## Eksempler
+
+### Uten overskrift
 
 Dersom du bruker `List.Heading` så legger vi automatisk på `aria-labelledby` på `List.Heading` og `List.Ordered/Unordered`
 for å gi skjermlesere beskjed om at `List.Heading` er en heading for `List`.
 
 Dersom du ikke legger til en heading kommer vi ikke til å legge til `aria-labelledby`.
 
-<Canvas of={ListStories.WithoutHeading} />
+<Canvas of={ListStories.UtenOverskrift} />
 
-### Avansert
+### Innrykk i lister
```

Comment:
```suggestion
### Lister i lister
```
Inntrykk er eit veldig vanskelig ord. Eg ville heller gått for "Lister i lister"